src/main/scala/com/databricks/sparkdl/DeepImageFeaturizer.scala

```diff
@@ -18,18 +18,21 @@ package com.databricks.sparkdl
 
 import java.nio.file.Paths
 
-import org.apache.spark.ml.image.ImageSchema
 import org.apache.spark.ml.Transformer
+import org.apache.spark.ml.image.ImageSchema
+import org.apache.spark.ml.image.ImageSchema.ocvTypes
 import org.apache.spark.ml.linalg.SQLDataTypes.VectorType
 import org.apache.spark.ml.linalg.Vectors
-import org.apache.spark.ml.param.{Param, ParamMap}
+import org.apache.spark.ml.param.{BooleanParam, Param, ParamMap}
 import org.apache.spark.ml.util.{DefaultParamsReadable, DefaultParamsWritable, Identifiable}
-import org.apache.spark.sql.{DataFrame, Dataset, Row}
 import org.apache.spark.sql.functions.{col, udf}
 import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.{DataFrame, Dataset, Row}
 import org.tensorflow.framework.GraphDef
-import org.tensorframes.{Shape, ShapeDescription}
 import org.tensorframes.impl.DebugRowOps
+import org.tensorframes.{Shape, ShapeDescription}
```

Comment:
Why the re-ordering of all the imports?

Reply:
result Idea's optimize imports. I guess it puts them in alphabetical order.